docs/_includes/layouts/pages/element.11ty.ts

```diff
@@ -167,7 +167,8 @@ export default class ElementsPage extends Renderer<Context> {
 
   async #renderOverviewPage(content: string, ctx: Context) {
     const description = ctx.doc.docsPage.description ?? ctx.doc.description ?? '';
-    return html`${!ctx.doc.planned ? '' : html`
```

Comment:
instead of editing this file, please make the preamble markdown content in the element docs, and remove the `preamble` flat from `ElementDocsPageBasicData`, as well as the preamble text from `repoStatus`.

That content can be a persistent toasted alert, or an inline alert, on the overview page.

Reply:
et voila: [491f78b](https://github.com/RedHat-UX/red-hat-design-system/pull/2171/commits/491f78b770555c3db1501f458b23f307d405c14f)

![image](https://github.com/user-attachments/assets/36d44650-3cd0-4b0a-8342-c285d789d0b5)